.github/workflows/preview-closed.yml

```diff
@@ -0,0 +1,34 @@
+# file: .github/workflows/preview-closed.yaml
+on:
+  pull_request:
+    types:
+      - closed
+jobs:
+  closed:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Context
+        uses: okteto/context@latest
+        with:
+          url: ${{secrets.OKTETO_URL}}
+          token: ${{ secrets.OKTETO_TOKEN }}
+      - name: Get Preview Endpoints
+        id: preview_urls
+        run: |
+          URLS=$(okteto preview endpoints pr-${{ github.event.number }}-${{ github.actor }} -o json | jq -c)
+          echo "::set-output name=preview_urls::$URLS"
+      - uses: ryanbas21/update-cors-am-github-action@latest
+        with:
+          AM_URL: ${{ secrets.AM_URL }}
+          USERNAME: ${{ secrets.AM_USERNAME }}
+          PASSWORD: ${{ secrets.AM_PASSWORD }}
+          REALM_PATH: ${{ secrets.AM_REALM }}
+          ORIGINS: ${{ steps.preview_urls.outputs.preview_urls }}
+          COOKIE_NAME: ${{ secrets.COOKIE_NAME }}
+          REDIRECTION_URIS: ${{ secrets.REDIRECTION_URIS }}
+          REMOVE: true
+          CORS_CONFIG_NAME: 'org-ui'
```

Comment:
Are we creating a CORS config called 'org-ui', or expecting it to be there? I remember that at some point the default was no longer named 'org-ui'. Maybe it has been reverted now, but we might be better off creating our own for the previews, if we are not already doing so.